content/blog/2021/06/2020-06-03-digester-removal.adoc

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
publish date needs to be updated before merge

Reply:
:+1: done